lib/instrumentation/dropped-span-stats.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+const LIMIT_STATS = 128
+class DroppedSpanStats {
+  constructor () {
+    this.statsMap = new Map()
+  }
+
+  captureDroppedSpan (span) {
+    const resource = span && span._destination && span._destination.service && span._destination.service.resource
+    if (!resource || !span._exitSpan || this.statsMap.size >= LIMIT_STATS) {
```

Comment:
Sorry for this to come late in the review process. I think that checking the `statsMap.size` against the limit here is in the wrong place. If we hit the 128 size limit, with the current code we will stop incrementing `stats.duration.*` values *for existing entries*.